modules/agent-host-roles.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="agent-host-roles_{context}"]
+= Host roles
+
+Each host in the cluster is assigned a role of either `master` or `worker`.
+You can define the role for each host in the `agent-config.yaml` file by using the `role` parameter.
+If you do not assign a role to the hosts, the roles will be assigned at random during installation.
+
+It is recommended to explicitly define roles for your hosts.
+
+The `rendezvousIP` must be assigned to a host with the `master` role, whether you explicitly assign the host role or whether the Agent-based Installer assigns the role.
```

Comment:
I might rephrase to:
"The `rendezvousIP` must be assigned to a host with the `master` role. This can be done manually or by allowing the Agent-based Installer to assign the role."

The two "whether's" sound a little funny to me. Just my opinion :)

Reply:
I think this reword sounds good, thanks